adapters/Vungle/VungleAdapter/Bidding/GADMediationVungleRewardedAd.m

```diff
@@ -167,20 +167,18 @@ - (void)adNotAvailable:(nonnull NSError *)error {
     return;
   }
   _adLoadCompletionHandler(nil, error);
-  [GADMAdapterVungleBiddingRouter.sharedInstance removeDelegate:self];
 }
 
 - (void)trackClick {
   [_delegate reportClick];
 }
 
 - (void)rewardUser {
-  id<GADMediationRewardedAdEventDelegate> strongDelegate = _delegate;
```

Comment:
@ManojBudumuru-vungle @stanley-vungle Can you remind me again why is this removed? If I remember correctly, you have verified that `_delegate` is already strong?

Reply:
Because it is the same thing so there was no need to create "strongDelegate". We can just use _delegate.